tajo-core/src/test/java/org/apache/tajo/engine/query/TestWindowQuery.java

```diff
@@ -462,4 +462,148 @@ public final void testMultipleWindow() throws Exception {
       executeString("DROP TABLE multiwindow PURGE");
     }
   }
+
+  @Test
+  public final void testWindowFrame1() throws Exception {
```

Comment:
The tests ```testWindowFrame1```, ```testWindowFrame2```, ```testWindowFrame3```, and ```testWindowFrame4``` look good, but it would be better if they are written in our test writing convention. That is separating java codes, test queries, and results.